src/test/java/com/codeofus/rent_a_park/PersonControllerTests.java

```diff
@@ -0,0 +1,66 @@
+package com.codeofus.rent_a_park;
+
+import com.codeofus.rent_a_park.controllers.PersonController;
+import com.codeofus.rent_a_park.dtos.ParkingMapper;
+import com.codeofus.rent_a_park.dtos.PersonDto;
+import com.codeofus.rent_a_park.models.Person;
+import com.codeofus.rent_a_park.repositories.PersonRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
isn't the random port default behaviour? if it is, we don't need to specify it, if it's not, sorry :)

Reply:
Default port is 8080

Reply:
Okay